### ISSUE TITLE
Delte record Will send Message to  destination ?

### ISSUE THREAD

**Issue by rongfengliang** [castledio/castled#48]
Current from source code  I seen than castled use except do diff for  pg like datawarehouse . But I didn't see any operator 

for  records that had been deleted from source datawarehouse . maybe am wrong?

**Comment by aruntcet2010** [comment]
@rongfengliang yes currently we do not support deletes from any of the existing warehouse connectors. But that can be easily supported if you want. Currently we do A except B, where A is the query result and B is the snapshot table. If you do B except A, you can get deletes as well for your warehouse.

Also, please feel free to raise concerns directly on our discord community as well.You will get speedy replies there.

**Comment by mkstayalive** [comment]
HI @rongfengliang, may I know what is the use case that you are trying to handle? What is the destination that you are syncing to and why is handling deletes important there?

**Comment by rongfengliang** [comment]
I Know that castled support Postgres .  current we use Greenplum as our  olap+oltp.  data store . for oltp we may delete record and we also want record  status can be passed to the destination  . so  the destination can use record status do other 
things (update  data status or delete  some data)
